cache/default.go

```diff
@@ -0,0 +1,76 @@
+package cache
+
+import (
+	"fmt"
+	"time"
+)
+
+// NewDefaultCache returns an instance of the default cache.
+func NewDefaultCache() *DefaultCache {
+	return &DefaultCache{items: make(map[string]*Item)}
+}
+
+// DefaultCache is the default implementation of the cache interface.
+type DefaultCache struct {
+	items map[string]*Item
+}
+
+// Get returns the item
+func (c *DefaultCache) Get(key string) (item *Item, err error) {
+	item, ok := c.items[key]
```

Comment:
This should probably check that the item hasn't expired, just in case get is called slightly before the next iteration of ExpireKeys